client/hosting-overview/components/plan-card.tsx

```diff
@@ -0,0 +1,119 @@
+import { PlanSlug } from '@automattic/calypso-products';
+import { ProgressBar, Button, Card, PlanPrice } from '@automattic/components';
+import { usePricingMetaForGridPlans } from '@automattic/data-stores/src/plans';
+import { formatCurrency } from '@automattic/format-currency';
+import { Icon, cloud } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import { FC } from 'react';
+import { useSelector } from 'react-redux';
+import QuerySitePlans from 'calypso/components/data/query-site-plans';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import useCheckPlanAvailabilityForPurchase from 'calypso/my-sites/plans-features-main/hooks/use-check-plan-availability-for-purchase';
+import { getCurrentPlan } from 'calypso/state/sites/plans/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+const PlanCard: FC = () => {
+	const moment = useLocalizedMoment();
+	const site = useSelector( getSelectedSite );
+	const sitePlan = site?.plan;
+	const planName = sitePlan?.product_name_short ?? '';
+	const planSlug = ( sitePlan?.product_slug || '' ) as PlanSlug;
+	const plan = useSelector( ( state ) => getCurrentPlan( state, site?.ID ) );
+	const pricing = usePricingMetaForGridPlans( {
+		coupon: undefined,
+		planSlugs: [ planSlug ],
+		selectedSiteId: site?.ID,
+		storageAddOns: null,
+		useCheckPlanAvailabilityForPurchase,
+	} );
+	const translate = useTranslate();
+
+	// TODO: Replace with real data using <PlanStorage /> component
+	const usedGigabytes = 47;
+	const availableUnitAmount = 50;
+
+	return (
+		<>
+			<QuerySitePlans siteId={ site?.ID } />
+			<Card className="hosting-overview__card">
+				<div>
+					<div className="hosting-overview__plan-card-header">
+						<h3 className="hosting-overview__plan-card-title">{ planName }</h3>
+
+						<Button
+							className="hosting-overview__link-button"
+							plain
+							href={ `/plans/${ site?.slug }` }
+						>
+							{ translate( 'Manage plan' ) }
+						</Button>
+					</div>
+					<PlanPrice
+						className="hosting-overview__plan-price"
+						currencyCode={ plan?.currencyCode }
+						displayPerMonthNotation
+						isSmallestUnit
+						rawPrice={ pricing?.[ planSlug ].originalPrice.monthly }
+					/>
+					<div className="hosting-overview__plan-info">
+						{ translate( '{{span}}%(rawPrice)s{{/span}} billed annually.', {
```

Comment:
I suspect that you'll want to clarify that this price excludes tax, otherwise might lead to some confused or aggrieved customers. For reference, here's what the Purchases section does! :) 

https://github.com/Automattic/wp-calypso/blob/6a1c49fe285bab7657dd409ebcd386a1ff68f2a4/client/me/purchases/purchase-item/index.jsx#L80-L83

Reply:
Good point, addressed on https://github.com/Automattic/wp-calypso/pull/89476/commits/7e86a9e14afa1e1f4adfa5e60e44641405e6d976. Thanks!